app/views/admin/loans/questionnaires/_questionnaire_group.html.slim

```diff
@@ -50,10 +47,10 @@
 
         / Recurse if group, else render
         - if question.data_type == 'group'
-          - if (filtered_children = question.children_applicable_to(@loan)).present?
+          - if (question.children).present?
             = render("admin/loans/questionnaires/questionnaire_group",
-              f: f, response_set: response_set, roots: filtered_children,
-              parents: parents + [question], indices: indices + [idx + 1])
+              f: f, response_set: response_set, roots: question.children,
+              parents: parents + [question])
```

Comment:
This can be just 2 lines now maybe?

Reply:
So this file is still kind of challenged but it's getting better. Going from here to something happier will be easier now.